src/game.cpp

```diff
@@ -4214,7 +4224,7 @@ void Game::updateFrame(ProfilerGraph *graph, RunStats *stats,
 	if (draw_control->range_all) {
 		runData->fog_range = 100000 * BS;
 	} else {
-		runData->fog_range = 0.9 * draw_control->wanted_range * BS;
+		runData->fog_range = draw_control->wanted_range * BS;
```

Comment:
This is (and should be, IMHO) fixed in #4882.
@Rogier-5 are you planning on closing #4882 in favor of this?